client_app/src/authentication/getAuthHeader.js

```diff
@@ -2,10 +2,10 @@ import getToken from "./getToken";
 
 export default function getAuthHeader() {
   const token = getToken();
-  if (token) {
-    return {
-      "Content-Type": "application/json",
-      Authorization: token,
-    };
-  }
+  if (!token) return undefined;
+
+  return {
+    "Content-Type": "application/json",
+    Authorization: `Bearer ${JSON.parse(token)}`, //  Cleaned format
```

Comment:
```suggestion
    Authorization: token`, 
```
I think this like will break our login functionality. The server expects it in a certain format.